doc/sphinx/glossary.rst

```diff
@@ -0,0 +1,36 @@
+Glossary
+========
+
+.. glossary::
+
+   bus
+      The communication backbone, providing `publish` and `listen` operations.
+      The bus takes care of sequencing all messages within the sysem and
+      serialization via logging facility.
+
+   driver
+      Implementation of common nodes like TCP communication with outer world.
+
+   link
+      Directed connection of two nodes. The source node generates messages by
+      `publish` call and the destinaion node accepts them via `listen` call.
+
+   logfile
+      Binary file containing message data with their source and timestamp.
+
+   node
+      The basic unit attached to `bus`. It has fixed set of inputs and outputs.
```

Comment:
I think I don't like having both `node` and `driver`. Can we have just `node`?

Reply:
Node is "everything" while "driver" is very specific to HW ... somehow I still see value using driver.

Reply:
I don't say it often but when I do, I do mean it.

Reply:
Sorry, I already overlooked/forgot this bit :(.